tensorflow/python/ops/math_grad.py

```diff
@@ -767,6 +767,10 @@ def _SparseMatMul(t1, t2, out_dtype,
 def _FloorGrad(_, unused_grad):
   return [None]
 
+@ops.RegisterGradient("Rint")
```

Comment:
NoGradient caused all of our researchers to throw a hissy-fit so we renamed it NotDifferentiable (marginally better for how it was used in most places, but w/e), which is still not really the same as "ZeroGradient".  I'd be fine with adding a ZeroGradient somewhere that is also the same, but frankly this change is also good, just with a comment saying something like "Propagate zeros in the gradient".